hack/preview-template.env

```diff
@@ -80,8 +80,17 @@ export DEPLOY_ONLY=""
 export DOCKER_IO_AUTH=""
 
 ## Pipelines as Code integration
-### Instructions for GitHub APP creation - https://pipelinesascode.com/docs/install/github_apps/#setup
+### Instructions for PaC GitHub application creation - https://pipelinesascode.com/docs/install/github_apps/#setup
 ### Webhook url, webhook secret is managed by preview.sh
 ### pipelines-as-code-secret is created by preview.sh
-export GITHUB_APP_PRIVATE_KEY= # Base64 encoded private key of the GitHub APP
-export GITHUB_APP_ID= # Application ID
+export PAC_GITHUB_APP_PRIVATE_KEY= # Base64 encoded private key of the GitHub APP
+export PAC_GITHUB_APP_ID= # Application ID
+
+# GitHub webhook integration (alternative to the GitHub PaC application)
+# See https://pipelinesascode.com/docs/install/github_webhook/#setup-git-repository for the required token permissions
+# MY_GITHUB_TOKEN is used as fallback
+export PAC_GITHUB_TOKEN=
+
+# GitLab webhook integration
+# See https://pipelinesascode.com/docs/install/gitlab/#create-gitlab-personal-access-token for the required token permissions
+export PAC_GITLAB_TOKEN=
```

Comment:
would it be worth to make a note about a possibility to configure PaC to a readme?

Reply:
Sure. Done